cwl/src/main/scala/cwl/Tool.scala

```diff
@@ -115,6 +117,20 @@ trait Tool {
         _.select[SchemaDefRequirement].toList
       }.headOption.getOrElse(SchemaDefRequirement())
 
+      lazy val localizationOptionalMetaObject: MetaValueElement = MetaValueElementObject(
+        Map(
+          "localization_optional" -> MetaValueElementBoolean(true)
+        )
+      )
+
+      // If input dir min == 0 that's a signal not to localize the input files
```

Comment:
Give credit where credit's due. From what I understand this isn't in the official spec, so please list for what group this extra code is being added. If it does happen to be true spec, then please mention so.

Reply:
As it is part of support for a specific hint, if it's somehow feasible to codify that in a way other than a comment that'd be 💯